docs/source/blocks/block-style-wrapper.md

```diff
@@ -236,6 +236,83 @@ You need to manually add the above code in your view component block code to ben
 The styles in the block edit component are injected automatically into the blocks engine editor wrappers, so you don't have to take any action.
 ```
 
+## Nested custom CSS properties
+
+Given this block enhancer:
+
+```js
+  schema.properties.styles.schema.fieldsets[0].fields = [
+    ...schema.properties.styles.schema.fieldsets[0].fields,
+    'theme',
+  ];
+
+  schema.properties.styles.schema.properties['theme'] = {
+    widget: 'color_picker',
+    title: "Theme",
+    colors,
+    default: defaultBGColor,
+  };
+```
+
+It will generate this values for the StyleWrapper to use:
```

Comment:
Is there a better term than "values" for the thingie below?

If not, this fixes the grammar.

```suggestion
It will generate these values for the `StyleWrapper` to use:
```

Reply:
I didn't find ones better :(